README.md

```diff
@@ -1 +1,96 @@
-# terraform-azurerm-rg
+<!-- BEGIN_TF_DOCS -->
+## About the Module
+This Terraform module creates an Azure Resource Group with the specified configuration.
+
+## How to Use
+To use this module, include the following code in your Terraform configuration:
+```
+module "resource_group" {
+  source = "dasmeta/rg/azure"
+
+  name = "my-resource-group"
+  location = "eastus"
+  tags = {
+    environment = "dev"
+  }
+}
+
+```
+## Requirements
+
+| Name                                                                      | Version |
+| ------------------------------------------------------------------------- | ------- |
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.2  |
+| <a name="requirement_azurerm"></a> [azurerm](#requirement\_azurerm)       | >= 1.32 |
+
+## Providers
+
+| Name                                                          | Version |
+| ------------------------------------------------------------- | ------- |
+| <a name="provider_azurerm"></a> [azurerm](#provider\_azurerm) | >= 1.32 |
+
+## Modules
+
+No modules.
+
+## Resources
+
+| Name                                                                                                                        | Type     |
+| --------------------------------------------------------------------------------------------------------------------------- | -------- |
+| [azurerm_resource_group.rg](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_group) | resource |
+
+
+## Inputs
+
+| Name                                                                  | Description                                                       | Type       | Default         | Required |
+| --------------------------------------------------------------------- | ----------------------------------------------------------------- | ---------- | --------------- | :------: |
+| <a name="input_rg_location"></a> [rg\_location](#input\_rg\_location) | The Azure Region where the Resource Group should exist.           | `string`   | `"West Europe"` |    no    |
```

Comment:
please enable pre-commit hooks and make sure it works, it should fix input_rg_name=>input_name, ...